src/commands/createNewProject/validateFunctionProjects.ts

```diff
@@ -152,19 +152,20 @@ async function promptToUpdateDebugConfiguration(fsPath: string): Promise<boolean
 
 async function verifyPythonVenv(projectLanguage: string | undefined, folderPath: string, actionContext: IActionContext): Promise<void> {
     if (projectLanguage === ProjectLanguage.Python) {
-        if (!await fse.pathExists(path.join(folderPath, funcEnvName))) {
+        const venvName: string | undefined = getFuncExtensionSetting(pythonVenvSetting, folderPath);
+        if (venvName && !await fse.pathExists(path.join(folderPath, venvName))) {
```

Comment:
Can't you just use `ensureVenv()`?

Reply:
Nah
1. That iterates through all possibilities - this is faster and only checks one
1. This logic has a message more specific to this scenario